rust/core-lib/src/line_offset.rs

```diff
@@ -12,15 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::linewrap::Lines;
+#![allow(clippy::range_plus_one)]
```

Comment:
again I would prefer us to fix the lint rather than allow it, unless there's a good reason not to.

Reply:
I've copied this line together with the `get_line_range` function from `view.rs`.

Reply:
(I hope this is a good reason.)